cytosnake/utils/cyto_paths.py

```diff
@@ -176,6 +188,24 @@ def get_project_dirpaths() -> dict:
         if _file.name in ignore_dirs:
             continue
         if _file.is_dir():
-            all_dirs[_file.name] = str(_file.absolute())
+            abs_path = str(_file.absolute())
+            all_dirs[_file.name] = abs_path
```

Comment:
`abs_path` variable stores `str(_file.absolute())` as a string object. JSON does not like it when you provide classes, since `pathlib.Path()` is a class object. 

In addition, `abs_path` was also reused a lot downstream.